client/src/Pages/Login.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import Navbar from "../Components/Navbar";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import i18 from 'i18next'
+import { Link } from "react-router-dom";
+
+const Login = () =>{
+
+    const [values,setValues] = useState({
+        gmail: '',
+        password: ''
+    })
+
+    const [isloggedIn, setisloggedIn] = useState(false);
+
+    useEffect(() => {
+        axios.get('http://localhost:3030/auth/status', { withCredentials: true })
+            .then(response => {
+                console.log('Auth status:', response.data);
+                setisloggedIn(response.data.loggedIn);
+            })
+            .catch(err => {
+                console.error('Error checking login status:', err);
+            });
+    }, [isloggedIn]);
+    
+    
+
+    const navigate = useNavigate();
+
+    const getLanguagePath = () => `/${i18.language}`
+
+    axios.defaults.withCredentials = true;
+
+    const handleSumbit = (e) =>
+        {
+            e.preventDefault();
+            if(isloggedIn)
+            {
+                alert('You are already logged in');
+                return;
+            }
+            if(values.gmail==='' || values.password==='')
+            {
+                alert('Make sure to fill both fields.')
+                return;
+            }
+            axios.post('http://localhost:3030/auth/login', values)
+            .then(res => 
+                {
```

Comment:
is this the first time `login` path?

Reply:
It is not in use as we are using `auth0` as the user management system.